backend/routes/friends.js

```diff
@@ -0,0 +1,79 @@
+const Parse = require('parse/node');
+const express = require("express")
+const router = express.Router()
+var cors = require('cors');
+
+router.use(cors())
+
+router.get('/users/:name', async (req, res) => {
+    try {
+        let query = new Parse.Query("_User")
+        query.equalTo("username", req.params.name)
+        const user = await query.first({useMasterKey : true})
```

Comment:
remind me again what master key is used for? I thought this was for the spotify API but i must be mistaken since that shouldnt be necessary for a friend request.

Reply:
master key is required when editing the user and session tables, but not for other queries. I've removed it from my other queries. I'm not sure if it's only required when editing or if it's required for editing and querying but I'll look into that.